webapp/src/main/webapp/templates/freemarker/body/partials/individual/propStatement-informationResourceInEditorship.ftl

```diff
@@ -12,10 +12,18 @@
 <#-- Use a macro to keep variable assignments local; otherwise the values carry over to the
      next statement -->
 <#macro showEditorship statement>
-    <#if statement.person??>
-        <a href="${profileUrl(statement.uri("person"))}" title="${i18n().editor_name}">${statement.personName}</a>
+    <#if statement.editor??>
+        <#if statement.subclass?? && statement.subclass?contains("vcard")>
+            <#if statement.editorName?replace(" ","")?length == statement.editorName?replace(" ","")?last_index_of(",") + 1 >
```

Comment:
Would it be possible to get a comment for what this is doing since it's a bit difficult to visually parse as a one-liner. If I'm understanding this correctly, this is trying to solve the additional comma issue I brought up above?